src/ContainerAppHelper.ts

```diff
@@ -49,21 +49,25 @@ export class ContainerAppHelper {
      * Creates an Azure Container App.
      * @param containerAppName - the name of the Container App
      * @param resourceGroup - the resource group that the Container App is found in
-     * @param environment - the Container App Environment that will be associated with the Container App
      * @param optionalCmdArgs - a set of optional command line arguments
+     * @param environment - the Container App Environment that will be associated with the Container App
      */
      public async createOrUpdateContainerAppWithUp(
         containerAppName: string,
         resourceGroup: string,
-        environment: string,
-        location: string,
-        optionalCmdArgs: string[]) {
+        optionalCmdArgs: string[],
+        environment?: string) {
         toolHelper.writeDebug(`Attempting to create Container App with name "${containerAppName}" in resource group "${resourceGroup}"`);
         try {
-            let command = `az containerapp up -n ${containerAppName} -g ${resourceGroup} --environment ${environment} -l northcentralusstage`;
+            let command = `az containerapp up -n ${containerAppName} -g ${resourceGroup} -l northcentralusstage`;
             optionalCmdArgs.forEach(function (val: string) {
                 command += ` ${val}`;
             });
+
+            if (!util.isNullOrEmpty(environment)) {
```

Comment:
Just want to be sure I get it: is there any case where we pass this environment in the current code? If not, then I would rather remove the logic until we know for sure we need it.

Reply:
We are not passing this `--environment` value to this method in our code. So, I removed it.

Reply:
Thanks